src/addedComponents/InsightsTimelineWidget/InsightsTimelineWidget.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+
+import { GrafanaTheme2, TimeRange } from '@grafana/data';
+import { usePluginComponent } from '@grafana/runtime';
+import { sceneGraph, SceneObject } from '@grafana/scenes';
+import { css } from '@emotion/css';
+import { useStyles2 } from '@grafana/ui';
+import { getMetricVariable } from 'utils/utils';
+import { MetricFunction } from 'utils/shared';
+
+export type AssertionSeverity = 'warning' | 'critical' | 'info';
```

Comment:
Note: can prob export these types from the Asserts [PR](https://github.com/grafana/asserts-app-plugin/pull/2013) and upgrade the plugin-types version here.